bundles/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -862,14 +861,25 @@ private JsonArray getJsonArray(String[] values) {
             // we have to strip ending "/" here because Kodi returns a not valid path and filename
             // "fanart":"image://http%3a%2f%2fthetvdb.com%2fbanners%2ffanart%2foriginal%2f263365-31.jpg/"
             // "thumbnail":"image://http%3a%2f%2fthetvdb.com%2fbanners%2fepisodes%2f263365%2f5640869.jpg/"
-            String encodedURL = URLEncoder.encode(StringUtils.stripEnd(url, "/"), StandardCharsets.UTF_8.name());
+            String encodedURL = URLEncoder.encode(stripEnd(url, '/'), StandardCharsets.UTF_8.name());
             return imageUri.resolve(encodedURL).toString();
         } catch (UnsupportedEncodingException e) {
             logger.debug("exception during encoding {}", url, e);
             return null;
         }
     }
 
+    private String stripEnd(String str, char suffix) {
```

Comment:
What about leaving out the generic part and using JDK methods:

```
    private String stripEnd(String str, String suffix) {
        if (str.endsWith(suffix)){
            return str.substring(0, suffix.length);
        } else {
            return str;
        }
```

Reply:
`stripEnd` isn't the same as `removeEnd`.

Reply:
Thanks for your review @martinvw. I did not change this one. It probably will not happen very often but I remember darkly the there was a reason for me to do it like this. The `stripEnd` method will not only remove on trailing character - it will remove all occurrences of the suffix at the end.

Reply:
> `stripEnd` isn't the same as `removeEnd`.

Then we should update the comment, it tells us we want to remove one:

>             // we have to strip ending "/" here because Kodi returns a not valid path and filename
>            // "fanart":"image://http%3a%2f%2fthetvdb.com%2fbanners%2ffanart%2foriginal%2f263365-31.jpg/"
>            // "thumbnail":"image://http%3a%2f%2fthetvdb.com%2fbanners%2fepisodes%2f263365%2f5640869.jpg/"